lib/fastlane/actions/pubu.rb

```diff
@@ -0,0 +1,83 @@
+module Fastlane
+  module Actions 
+    # To share this integration with the other fastlane users:
+    # - Fork https://github.com/KrauseFx/fastlane
+    # - Clone the forked repository
+    # - Move this integration into lib/fastlane/actions
+    # - Commit, push and submit the pull request
+
+    # noinspection ALL
+    class PubuAction < Action
+      def self.run(params)
+        url = ENV['PUBU_URL']
+        message = params[:message].to_s
+        pubuayload = buildPayload(message, params)
+        sendPubu(url, pubuayload)
+        raise "`PUBU_URL` lost  please set ENV first`".red unless url
+      end
+
+
+      def self.available_options
+        # Define all options your action supports.
+
+        # Below a few examples
+        [
+            FastlaneCore::ConfigItem.new(key: :message,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.